src/app/theme.ts

```diff
@@ -102,6 +102,7 @@ export default {
     modal: 12,
     navbar: 5,
     overlay: 3,
+    searchSidebar: 1,
```

Comment:
we want this to be above `contentNotifications`

Reply:
like the variable? or the actual index number?

Reply:
the index number. with what you have here, if you trigger the updates available it'll show up on top of the sidebar